.travis.yml

```diff
@@ -141,9 +142,10 @@ matrix:
       script:
         - export PATH=$(pwd)/mason_packages/.link/bin/:${PATH}
         - which llvm-cov
-        - curl -S -f https://codecov.io/bash -o codecov
-        - chmod +x codecov
-        - ./codecov -x "llvm-cov gcov" -Z
+        # upload coverage
+        - pip install --user codecov
```

Comment:
This moves to the `python` codecov client because:

  - bash one is broken due to travis changes: https://github.com/mapbox/node-cpp-skel/issues/158
  - node.js one does not correctly detect C++ coverage data (needs a bug report at https://github.com/codecov/codecov-node/issues)

Reply:
Thanks for the context!